cmd/ocm/create/cluster/cmd.go

```diff
@@ -385,7 +385,7 @@ func preRun(cmd *cobra.Command, argv []string) error {
 		return err
 	}
 
-	err = arguments.CheckIgnoredAutoscalingFlags(args.autoscaling, args.computeNodes)
+	err = arguments.CheckAutoscalingFlags(args.autoscaling, args.computeNodes)
```

Comment:
It's confusing that this is called after `promptAutoscaling` but before prompting for `computeNodes`, yet checks both of them...
But that's out of scope for this PR, I'll send a followup.